lib/screens/repository_screen/repository_screen.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:top_git_rep/design/app_localization.dart';
+import 'package:top_git_rep/design/app_theme_text.dart';
+import 'package:top_git_rep/design/dimensions.dart';
+import 'package:top_git_rep/models/search_result_item.dart';
+
+class RepositoryScreen extends StatelessWidget {
+  RepositoryScreen({required this.repository});
+
+  final List<Repository> repository;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+          title: const Text(AppLocalization.REPOSITORIES),
+          leading: GestureDetector(
+            child: IconButton(
+              icon: const Icon(Icons.arrow_back),
+              onPressed: () {
+                Navigator.pop(context);
+              },
+            ),
+          )),
+      // ),
```

Comment:
remove comments

my common approach for the comments is:

use comments to describe code
or
hide some code by commenting and leave todo, that would describe why that code is here